internal/explorer/models.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // DefaultExplorerURL is the default explorer graphql url
-const DefaultExplorerURL string = "https://tfchain.dev.threefold.io/graphql/"
+const DefaultExplorerURL string = "https://tfchain.dev.threefold.io/graphql/graphql"
```

Comment:
let's have the graphql configurable as well, we will need it for more networks with this one as a default

Reply:
with flag `--explorer`